recipes/log4cxx/all/conanfile.py

```diff
@@ -24,15 +24,33 @@ class Log4cxxConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "char_type": ["utf-8", "wchar_t", "unichar"],
+        "char_encoding": ["utf-8", "locale"],
+        "with_networking": [True, False],
+        "with_wchar_t": [True, False],
+        "with_fmt_layout": [True, False],
+        "with_odbc_appender": [True, False],
+        "with_multiprocess_rolling_file_appender": [True, False],
+        "with_smtp_appender": [True, False],
+        "with_qt": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
+        "char_type": "utf-8",
+        "char_encoding": "utf-8",
+        "with_networking": True,
+        "with_wchar_t": False,
+        "with_fmt_layout": False,
+        "with_odbc_appender": False,
+        "with_multiprocess_rolling_file_appender": False,
+        "with_smtp_appender": False,
+        "with_qt": False,
     }
```

Comment:
An `options_description` attribute would not hurt. E.g. it's not immediately particularly clear what the purpose of `char_type` vs `with_wchar_t` is.